torch/ao/quantization/fx/utils.py

```diff
@@ -972,11 +972,16 @@ def _activation_post_process_satisfies_dtype_config_constraints(
             activation_post_process: Union[ObserverBase, FakeQuantize],
             dtype_with_constraints: DTypeWithConstraints,
             debug_string: str) -> bool:
-        app_quant_min = getattr(activation_post_process, "quant_min", None)
-        app_quant_max = getattr(activation_post_process, "quant_max", None)
+        # For FakeQuantize, the attributes are set in the observer inside the FakeQuantize
+        if isinstance(activation_post_process, FakeQuantize):
+            observer = activation_post_process.activation_post_process
+        else:
+            observer = activation_post_process
```

Comment:
This is a small bug fix for https://github.com/pytorch/pytorch/pull/85200, which did not handle `FakeQuantize` correctly